index.js

```diff
@@ -5,31 +5,55 @@ module.exports = {
   extends: 'eslint:recommended',
   rules: {
     'arrow-parens': ['error', 'always'],
+    'prefer-const': ['error'],
+    'object-shorthand': ['error'],
+    'space-before-function-paren': [
+      'error',
+      {
+        anonymous: 'never',
+        named: 'never',
+        asyncArrow: 'always'
+      }
+    ],
     'jsx-quotes': ['error', 'prefer-single'],
-    'no-mixed-operators': ['error', {'groups': [['&&','||']]}],
-    indent: ['error', 2, {
-      SwitchCase: 1
-    }],
+    'no-mixed-operators': ['error', { groups: [['&&', '||']] }],
+    indent: [
+      'error',
+      2,
+      {
+        SwitchCase: 1
+      }
+    ],
     'linebreak-style': ['error', 'unix'],
     semi: ['error', 'always'],
     'no-use-before-define': ['error', 'nofunc'],
-    'no-unused-vars': ['error', {
-      // Per https://github.com/eslint/eslint/issues/1494#issuecomment-232890308.
-      argsIgnorePattern: 'next'
-    }],
-    quotes: ['error', 'single', {
-      // Let people use strings like "don't worry about single quotes".
-      avoidEscape: true
-    }],
+    'no-unused-vars': [
+      'error',
+      {
+        // Per https://github.com/eslint/eslint/issues/1494#issuecomment-232890308.
+        argsIgnorePattern: 'next'
+      }
+    ],
+    quotes: [
+      'error',
+      'single',
+      {
+        // Let people use strings like "don't worry about single quotes".
+        avoidEscape: true
+      }
+    ],
     // Enforce our whitespace styles around keywords.
-    'keyword-spacing': ['error', {
-      overrides: {
-        function: {
-          after: false
+    'keyword-spacing': [
+      'error',
+      {
+        overrides: {
+          function: {
+            after: false
+          }
```

Comment:
I liked having the whitespace for these rules consistent with the other rules :(